<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlDock_PluginTabWidget</class>
 <widget class="QWidget" name="ControlDock_PluginTabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>621</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QPushButton" name="button_addPluginDirectory">
     <property name="text">
      <string>Add directory</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>A
v
a
i
l
a
b
l
e</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QListWidget" name="list_pluginsAvailable">
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="button_enablePlugins">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: #bbb;
	border: 2px solid black;
	border-radius: 5px;
	background-image: url(':icons/icons/arrow_down.png');
	background-repeat: no-repeat;
	background-position: center;
	margin-right: 5px;
}
QPushButton:pressed {
	background-color: #eee;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="button_disablePlugins">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: #bbb;
	border: 2px solid black;
	border-radius: 5px;
	background-image: url(':icons/icons/arrow_up.png');
	background-repeat: no-repeat;
	background-position: center;
	margin-left: 5px;
}
QPushButton:pressed {
	background-color: #eee;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>E
n
a
b
l
e
d</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QListWidget" name="list_pluginsEnabled">
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
